Dockerfile_mbtci_java17

```diff
@@ -0,0 +1,93 @@
+FROM sapmachine:17
```

Comment:
is this file a 98% duplication of the java11 variant?
Can you think of a way to avoid this duplication?